third_party/openmetadata/Makefile

```diff
@@ -6,8 +6,18 @@ deploy-openmetadata:
 	helm repo add open-metadata https://helm.open-metadata.org/ || true
 	helm repo update || true
 	kubectl create ns ${INSTALLATION_NAMESPACE} || true
+ifeq ($(OPENSHIFT_INSTALLATION), true)
+	oc adm policy add-scc-to-user anyuid -z airflow -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user anyuid -z mysql -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user ibm-privileged-scc -z default -n ${INSTALLATION_NAMESPACE}
+	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pvc3.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
```

Comment:
Why do you replace namespaces? Do not specify any namespace in the yamls, and deploy to the correct namespace directly.

Reply:
we want to allow the user to edit `Makefile.env`, and choose a different namespace

Reply:
Why not wrapping this in a helm chart using templated release namespace as we are doing for other data catalogs?

Reply:
This is the best that we can do, given that we are using `OpenMetadata` helm charts (not our own) + patches to make those work with `kind` or `OpenShift`